code/modules/art/statues.dm

```diff
@@ -348,9 +348,12 @@ Moving interrupts
 	var/interrupted = FALSE
 	var/remaining_time = sculpting_time - (prepared_block.completion * sculpting_time)
 
-	var/datum/progressbar/total_progress_bar = new(user, sculpting_time, prepared_block )
+	var/datum/progressbar/total_progress_bar = new(user, sculpting_time, prepared_block)
 	while(remaining_time > 0 && !interrupted)
-		if(do_after(user,sculpting_period, target = prepared_block, progress = FALSE))
+		if(do_after(user, sculpting_period, target = prepared_block, progress = FALSE))
+			var/time_delay = !(remaining_time % 4)
```

Comment:
This is not what I meant, it's just the magic '4' number

Reply:
The time delay is not in `SECONDS`  or `MINUTES` it is every 4th loop through the `while()` loop.  That's what the modulo operator does with the way I'm using it here.

Reply:
Yes I know, I mean put the 4 on the object

Reply:
https://github.com/tgstation/tgstation/pull/72638/commits/4447d46078003bb9109f6d02b64600b51635a1b3